docs/workflow.md

```diff
@@ -0,0 +1,403 @@
+
+    1. HTTP Request - Verify that the `ServiceRequest` was successful in both OpenHIM Mediator & FHIR Resource. Navigate to the `Transaction Log` in the Admin Console. You should see three successful API calls, as in the image below:
+       ![](./images/instance-service-request.png)
+
+1.  Handle LTFU Task
+
+    1. CHT - Navigate to the `Tasks` tab. There should be an automatically created `Task` for the Patient. If it is not the case, sync data via `Sync now` option. The `Task` should look like in the image below:
+
+     <img src="./images/task.png" width="500">
+
+    1. CHT - Select an option (Yes or No) and submit the `Tasks`.
+    1. OpenHIM Admin Console - Verify that the Encounter creation was successful in both OpenHIM Mediator & FHIR Resource. Navigate to the `Transaction Log` in the Admin Console. You should see two successful API calls, one to `/mediator/encounter/` and one to `/fhir/Encounter/`, as in the image below.
+       ![](./images/instance-encounter.png)
+    1. If your callback URL test service was set up correctly, you should receive a notification from the mediator.
+
+## Resources
+
+The following [FHIR Resources](https://www.hl7.org/fhir/resource.html) are used to implement the flow above:
+
+- [Patient](https://www.hl7.org/fhir/patient.html)
+- [Encounter](https://build.fhir.org/encounter.html)
+- [Subscription](https://build.fhir.org/subscription.html)
+- [Organization](https://build.fhir.org/organization.html)
+- [Endpoint](https://build.fhir.org/endpoint.html)
+
+The payload samples in the documentation contain placeholder values you must replace with the actual content. To do so, replace the entire `${}` placeholder with the appropriate value. Be aware that some placeholder keys have the format `_\_IDENTIFIER` and refer to the value in the `Resource.identifier[0].value` field. These keys differ from the `_\_ID` placeholders used in the request, which refer to the `Resource.id` field. It is important to make this distinction, as using the wrong value may cause unexpected behavior in the system. Therefore, always ensure you use the right value in the right context to avoid errors.
+
+
+> **Note:** The payload only contains the required fields or a subset of the possible options. Please refer to the appropriate FHIR resource specifications to view all the available fields.
+
+
+### `ServiceRequest` Resource
+
+The FHIR `ServiceRequest` resource represents a request for a healthcare service to be performed, such as a diagnostic test or a treatment. It contains information about the requested service, including the **type of service**, the **patient** for whom the service is requested, the **date/time the service is requested**, and the **healthcare provider or organization** making the request. In the context of the LTFU workflow, this resource is used to request a CHW follow-up in the CHT.
+
+#### `POST ${MEDIATOR_ENDPOINT}/service-request`
```

Comment:
This is the only place in this documentation that we are using MEDIATOR_ENDPOINT instead of just using OPENHIM_ENDPOINT? is there a reason for this? Can we use normalize?
Also related to previous comments.